user_docs/en/userGuide.t2t

```diff
@@ -888,9 +888,15 @@ If input is not supported on a display which does have a braille keyboard, this
 ==== Expand to computer braille for the word at the cursor ====
 This option allows the word that is under the cursor to be displayed in non-contracted computer braille.
 
-==== Cursor Blink Rate ====
+==== Show Cursor ====
+This option allows the braille cursor to be turned on and off. It applies to all cursors including the selection indicator.

```

Comment:
nit: In the User Guide, each sentence should generally be on a separate line to make life easier for translators. Same applies to a later paragraph.